KubeArmor/core/k8sHandler.go

```diff
@@ -281,6 +282,34 @@ func (kh *K8sHandler) PatchDeploymentWithAppArmorAnnotations(namespaceName, depl
 	return nil
 }
 
+func (kh *K8sHandler) GetContainerImage(namespaceName, podname, containerId string) (string, error) {
```

Comment:
If k8s uses Containerd instead of Docker, I think this function may work incorrectly.